lib/atomic/organizations.ex

```diff
@@ -227,6 +227,28 @@ defmodule Atomic.Organizations do
     end
   end
 
+  @doc """
+  Verifies if an user is an admin or owner of an organization that is organizing an activity.
+
+  ## Examples
+
+      iex> verify_admin(user, departments)
+      true
+
+      iex> verify_admin(user, departments)
+      false
+
+  """
+  def verify_admin(_user, []), do: false
```

Comment:
```suggestion
  def is_admin?(_user, []), do: false
```
I prefer the old name 👀